docs/AboutUs.adoc

```diff
@@ -27,12 +27,12 @@ Responsibilities: UI
 
 '''
 
-=== Johnny Doe
-image::yijinl.jpg[width="150", align="left"]
-{empty}[http://github.com/yijinl[github]] [<<johndoe#, portfolio>>]
+=== Lai Liwen
+image::liwen.jpg[width="150", align="left"]
+{empty}[https://github.com/Livian1107[github]] [<<johndoe#, portfolio>>]
```

Comment:
Do we need to update the reference for portfolio?

Reply:
For now we can just let John Doe's portfolio be. Later when we make our own,  we can put the link over here